test/buy-tshirt.spec.ts

```diff
@@ -1,13 +1,14 @@
-/* eslint-disable protractor/no-describe-selectors */
-
-import { browser, by } from 'protractor';
+/* eslint-disable no-console */
+import { browser } from 'protractor';
+import {
+  SignInPage, InventoryPage, ShoppingCartPage, SummaryPage, CheckoutPage, CheckoutCompletePage
+} from '../src/page';
 
 const baseUrl = 'https://www.saucedemo.com/';
 
 describe('Buy a t-shirt', () => {
   beforeEach(async () => {
     jasmine.DEFAULT_TIMEOUT_INTERVAL = 120000;
-
     const userName = 'standard_user';
     const passwd = 'secret_sauce';
```

Comment:
delete these and firstName, lastName, zipCode
pass directly because you're using just one time